pkgs/by-name/cu/cue/package.nix

```diff
@@ -30,7 +31,7 @@ buildGoModule rec {
     "-X cuelang.org/go/cmd/cue/cmd.version=v${version}"
   ];
 
-  postInstall = ''
```

Comment:
I think we should be using `stdenv.hostPlatform.emulator buildPackages` here if available, e.g.
https://github.com/NixOS/nixpkgs/blob/49515b38849b3640f0ae8c67e2a3b1861db032a2/pkgs/by-name/ya/yandex-cloud/package.nix#L36-L38
https://github.com/NixOS/nixpkgs/blob/49515b38849b3640f0ae8c67e2a3b1861db032a2/pkgs/by-name/ya/yandex-cloud/package.nix#L47-L52

Reply:
> I think we should be using `stdenv.hostPlatform.emulator buildPackages` here if available, e.g.

Yes but no, since it brings the entire qemu into the build closure, which is not that justifiable for some shell completions.

Reply:
I think we should at least have an option to do that. It’s really weird to implicitly get a cut down version of the package just because build platform cannot execute code built for the host platform.

Reply:
Otherwise if shell completions are platform-independent, perhaps it would make sense to use `cue` from `buildPackages` instead to generate them.